### DISCUSSION TITLE
Why is createMany missing?

### DISCUSSION THREAD

**Discussion by a-random-lemurian** [prisma/prisma#20105] [Q&A] [answered]
Why does `.createMany()` not appear when I try to insert data into Prisma? I'm using SQLite and here is my code:

```ts
async ports(qty: number) {
      for (let i = 0; i < qty; i++) {
        await prisma.port.create({
          data: {
            name: DataGenerator.Port.makePortName()
          }
        });
      }
```

As you can see this function is meant to insert "ports" into Prisma on a massive scale hence the for loop. I want to use createMany for this but it seems as if it doesn't exist. How do I get createMany to appear?

`"@prisma/client": "^4.16.2"`

**Comment by a-random-lemurian** [comment, marked as answer]
Welp, turns out createMany is not supported by SQLite..

**Reply by janpio** [reply]
Yes unfortunately, issue is here: https://github.com/prisma/prisma/issues/10710

**Reply by a-random-lemurian** [reply]
For now as a work around I've just decided to put all my inserts into an array and then use `prisma.$transaction()`.
